microcosm_eventsource/stores/rollup.py

```diff
@@ -88,6 +88,12 @@ def search(self, **kwargs):
             for row in self._search_query(aggregate, **kwargs).all()
         ]
 
+    def search_first(self, **kwargs):
+        results = self.search(**kwargs)
+        if results:
+            return results[0]
+        return None
```

Comment:
raise?

Reply:
microcosm-postgres doesn't seem to raise on no result, I think we shouldn't
https://github.com/globality-corp/microcosm-postgres/blob/develop/microcosm_postgres/store.py#L184